pkg/frontend/mysql_cmd_executor.go

```diff
@@ -1663,7 +1663,11 @@ func (cwft *TxnComputationWrapper) Compile(requestCtx context.Context, u interfa
 
 	cwft.proc.UnixTime = time.Now().UnixNano()
 	txnHandler := cwft.ses.GetTxnHandler()
-	cwft.proc.TxnOperator = txnHandler.GetTxn()
+	if _, ok := cwft.stmt.(*tree.Load); !ok {
```

Comment:
Wait -- what is this ...     Now you access its private member?

Reply:
oh, I' ll use a public function to get the member